src/client/firefox/commands.js

```diff
@@ -262,7 +262,7 @@ function pauseGrip(func: Function): ObjectClient {
 
 async function fetchSources() {
   const { sources } = await threadClient.getSources();
-  return sources.map(createSource);
+  return sources.map(source => createSource(source, debuggerClient));
```

Comment:
perhaps a slightly different signature. 


```js
const supportsWasm = !!debuggerClient.mainRoot.traits.wasmBinarySource;
createSource(source, { supportsWasm }))
```

The rationale is that when devtools moves to github it will be easier to add a `supportsWasm` function in debuggerClient.